packages/react-tabster/package.json

```diff
@@ -17,10 +17,11 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start-test": "just-scripts jest-watch",
     "storybook": "start-storybook",
-    "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest -u"
+    "test": "jest --passWithNoTests",
```

Comment:
There are already e2e tests for this package

Reply:
not sure I follow the comment with its context ? 👀

Reply:
the migration script adds `jest.config` which will fail CI because there are no jest unit tests. However there are E2E tests for cypress in this repo and in with puppeteer in the tabster repo.

Do you have any better suggestion for this ?

Reply:
I see! lets keep it as is. 

in future when we'll leverage nx executors, jest has this turned on by default AFAIR. thx again 🙌